Dockerfile-build

```diff
@@ -155,7 +155,7 @@ RUN bash -c 'if [ `arch` = "ppc64le" ]; then \
 #
 # Install Python requirements
 #
-RUN pip install numpy==1.14.5 scipy==1.1.0 setuptools==39.0.1
+RUN pip install numpy==1.15.1 scipy==1.1.0 setuptools==39.0.1
```

Comment:
Was there a need to update numpy?  Just want to avoid mismatch with DAI if possible for now till we figure out.

Reply:
I also recollect that there were some issues with complete conda package dependency tree when numpy was increased. Though that may have changed. 

our build process should ensure that both pip and conda packages can be created but i believe that would have been taken care. Just adding note as caution.

Reply:
There's a discrepancy between version in Docker file and https://github.com/h2oai/h2o4gpu/blob/master/src/interface_py/requirements_runtime.txt. It results in numpy=1.14 get installed and uninstalled to install version 1.15. Using the same version can speed-up CI a bit.